graylog2-web-interface/src/components/streams/StreamMetaData.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import CollapsibleStreamRuleList from 'components/streamrules/CollapsibleStreamRuleList';
+import { Pluralize } from 'components/common';
+import { Button } from 'components/graylog';
+
+import StreamThroughput from './StreamThroughput';
+
+const StreamMetaDataWrapper = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const StreamMetaData = ({ isDefaultStream, stream, streamRuleTypes, permissions }) => {
+  let verbalMatchingType;
+  const [expanded, setExpanded] = useState(false);
+  const toggleText = expanded ? 'Hide' : 'Show';
+
+  if (stream.is_default) {
+    return 'The default stream contains all messages.';
+  }
+  if (stream.rules.length === 0) {
+    return 'No configured rules.';
+  }
+
+  switch (stream.matching_type) {
+    case 'OR': verbalMatchingType = 'at least one'; break;
+    default:
+    case 'AND': verbalMatchingType = 'all'; break;
+  }
+
+  const _onHandleToggle = () => {
+    setExpanded(!expanded);
+  };
+
+  return (
+    <StreamMetaDataWrapper>
+      <StreamThroughput streamId={stream.id} />.
+
+      <span>
+        &nbsp;Must match {verbalMatchingType} of the {stream.rules.length} configured stream&nbsp;
+        <Pluralize value={stream.rules.length} plural="rules" singular="rule" />.
+      </span>
+
+      {!isDefaultStream && (
+        <>
+          <Button bsStyle="link"
+                  bsSize="xsmall"
+                  onClick={_onHandleToggle}>
+            {toggleText} stream rules
+          </Button>
+
+          <CollapsibleStreamRuleList key={`streamRules-${stream.id}`}
```

Comment:
With the last changes this doesn't look right:
<img width="1647" alt="Screenshot 2020-02-03 at 11 26 18" src="https://user-images.githubusercontent.com/716185/73646000-c5656d80-4678-11ea-8d68-16df96457a26.png">

Reply:
![image](https://user-images.githubusercontent.com/122591/73657473-f71a0c80-4657-11ea-9c1c-f40bc0e8d083.png)